triton-vm/Cargo.toml

```diff
@@ -31,7 +31,7 @@ default-features = false
 
 [dependencies]
 twenty-first = "0.10"
-triton-opcodes = "0.11"
+triton-opcodes = {path = "../triton-opcodes"}
```

Comment:
This should be changed back in correspondence with releasing a new of `triton-opcodes`.